packages/shared/routes/onboarding/OnboardingCustomNetworkView.svelte

```diff
@@ -1,18 +1,49 @@
 <script lang="typescript">
     import { OnboardingLayout, Text, Button, Spinner, NodeConfigurationForm } from 'shared/components'
     import { localize } from '@core/i18n'
-    import { INode } from '@core/network'
+    import { INode, NetworkType } from '@core/network'
     import { appRouter } from '@core/router'
+    import { showAppNotification } from '@lib/notifications'
+    import { deleteAccountsAndDatabase, destroyProfileManager, getNodeInfo } from '@core/profile-manager'
+    import { initProfileManagerFromNewProfile, newProfile } from '@contexts/onboarding'
 
     let nodeConfigurationForm: NodeConfigurationForm
     let node: INode
     let isBusy = false
+    let formError = ''
 
     function onBackClick(): void {
         $appRouter.previous()
     }
-    function onSuccess(): void {
-        $appRouter.next()
+
+    async function handleAddNode(): Promise<void> {
+        isBusy = true
+        try {
+            await nodeConfigurationForm.validate({
+                validateUrl: true,
+                checkSameNetwork: false,
+                checkNodeInfo: false,
+                validateClientOptions: false,
+            })
+            await initProfileManagerFromNewProfile($newProfile.networkProtocol, NetworkType.PrivateNet, node)
+            await getNodeInfo(node.url)
```

Comment:
We can set check node info, and remove the local call.

Reply:
not possible here, because to check the node info, we need a profilemanager. And here we init it after the `validate`